core/src/main/scala/akka/kafka/internal/KafkaConsumerActor.scala

```diff
@@ -270,14 +272,9 @@ import scala.util.control.NonFatal
       }
       commitRefreshing.assignedPositions(assignedOffsets.keySet, assignedOffsets)
 
-    case Commit(offsets) if rebalanceInProgress =>
-      rebalanceCommitStash ++= offsets
-      rebalanceCommitSenders = rebalanceCommitSenders :+ sender()
-
     case Commit(offsets) =>
-      commitRefreshing.add(offsets)
-      val replyTo = sender()
-      commit(offsets, replyTo ! _)
+      commitAggregate ++= offsets
+      commitSenders = commitSenders :+ sender()
```

Comment:
A thing to consider would be to request an extra poll by setting up a "delayed poll in flight" as the `RequestMessages` do.